docs/modules/development/pages/rest/health_rest.adoc

```diff
@@ -0,0 +1,43 @@
+
+= Health Rest Service
+
+Service that allows health rest calls.
+
+Currently implemented on Minion and Sentinel by default.
+
+Examples:
+
+* /rest/health
+* /rest/health/probe
+* /rest/health?tag=local
+* /rest/health/probe?tag=local
+
+== GETs (Reading Data)
+
+[options="header", cols="5,10"]
+|===
+| Resource             | Description
```

Comment:
```suggestion
| Resource            
| Description
```

Reply:
Table syntax is to have each cell on a separate line, a blank line between rows. (p.s. I hate tables. :-))